testgrid/config/yaml2proto/testdata/large.yaml

```diff
@@ -0,0 +1,809 @@
+default_test_group:
```

Comment:
Please stay consistent in your filenames. Make all files lower case or all files capitalized.